### PR TITLE
Add query option to api doc

### PR DESCRIPTION
**Pull request by JeffreyDevloo**

### CODE REVIEW

File: Administration/usingtheapi/README.md
Comment (kvanhijf) [inline Administration/usingtheapi/README.md]: less than

**Review by kvanhijf — CHANGES_REQUESTED** [review]

**Comment by wimpers** [comment]
Do not merge this PR unless we a package with this version in the OSE repo.

**Review by sjovdnbos — APPROVED** [review]

**Pull request merged**
